fortex/health/processors/scispacy_processor.py

```diff
@@ -91,18 +91,37 @@ def _process(self, input_pack: DataPack):
 
             else:
                 for item in doc._.hearst_patterns:
+
+                    (
```

Comment:
I think we can skip these parenthesis (here and in the return statement), since it just complicates the readability in this case, and doesn't add to functionality. Just a minor suggestion, upto if you want to make that change. Also, maybe we can do the index finding more elegantly. Since, in a scenario with multiple hyponyms for a bigger text chunk, we traverse the whole text chunk for each detected hyponym, which may not be performant. We can do that in the next iteration too.

Reply:
this is done by black actually